imap_data_access/cli.py

```diff
@@ -252,10 +252,16 @@ def main():  # noqa: PLR0915
         help="Descriptor of the product (raw, burst, etc.)",
     )
     query_parser.add_argument(
-        "--start-date", type=str, required=False, help="Start date in YYYYMMDD format"
+        "--start-date",
+        type=str,
+        required=False,
+        help="Start date for files in YYYYMMDD format",
     )
     query_parser.add_argument(
-        "--end-date", type=str, required=False, help="End date in YYYYMMDD format"
+        "--end-date",
+        type=str,
+        required=False,
+        help="End date for a range of file start dates in YYYYMMDD format",
```

Comment:
```suggestion
        help="End date for a range of file timestamps in YYYYMMDD format",
```
Maybe you can take out the concept of a "file start date" from these messages to make them less confusing?